remoteworkitem/remoteworkitem_test.go

```diff
@@ -1,18 +1,26 @@
-// +build unit
-
 package remoteworkitem
 
 import (
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net/http"
 	"testing"
 
+	"github.com/almighty/almighty-core/resource"
 	"github.com/almighty/almighty-core/test"
+	"github.com/stretchr/testify/assert"
 )
 
-func provideRemoteGithubData() ([]byte, error) {
-	url := "https://api.github.com/repos/almighty/almighty-core/issues/131"
+const (
+	GithubIssueWithAssignee    = "http://api.github.com/repos/almighty-test/almighty-test-unit/issues/2"
+	GithubIssueWithoutAssignee = "http://api.github.com/repos/almighty-test/almighty-test-unit/issues/1"
+	JiraIssueWithAssignee      = "http://jira.atlassian.com/rest/api/latest/issue/JRA-9"
+	JiraIssueWithoutAssignee   = "http://jira.atlassian.com/rest/api/latest/issue/JRA-10"
```

Comment:
Changed my mind : Added tests for the 'without' assignee cases, where all we did was skip testing the assignee field.